packages/m2m-gateway/src/utils/errorMappers.ts

```diff
@@ -40,6 +40,16 @@ export const getEServiceTemplateVersionErrorMapper = (
     .with("eserviceTemplateVersionNotFound", () => HTTP_STATUS_NOT_FOUND)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const unsuspendEServiceTemplateVersionErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with(
+      "eserviceTemplateVersionNotInSuspendedState",
+      () => HTTP_STATUS_CONFLICT
+    )
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
```

Comment:
Unneeded, see: https://github.com/pagopa/interop-be-monorepo/pull/2407#discussion_r2378242011

```suggestion
```